### ISSUE TITLE
Plugin should adhere to language level for recommendations (array_column usage)

### ISSUE THREAD

**Issue by fruitl00p** [kalessil/phpinspectionsea#1508]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate) |
| Language level | PHP 5.5                        |
| CLI Interpreter version | PHP 7+ |

I'm running into an issue where PHPStorm is using a 7.4 CLI interpreter but the IDE is still on PHP 5.5 Language level. (due to multiple deployments et al)

This seems to result in a false-positive for the array_column usage when iterating over an array of objects:
```php
array_map(static function(\stdClass $item) {
    return $item->id;
}, $items)))
```
Results in a suggestion for `array_column`-usage but as the language level is still PHP5.5 the `array_column`-usage isn't possible (yet) (i.e. ```array_column($items, 'id')```

Does EA check for language level in this case at all or is this a bug in the array_column-specific proposed solution?

Thanks!

**Comment by kalessil** [comment]
@fruitl00p: checked the inspection checks language level, and if it's below 5.5 no warning gets emitted.

**Comment by fruitl00p** [comment]
Shouldn't it be higher than 5.5 though? IIRC the object variant is only available since 7+?

**Comment by kalessil** [comment]
Oh, I understand now. You are right, processing object needs 7.0+

**Comment by kalessil** [comment]
Fixed!
